lib/web_ui/test/text_editing_test.dart

```diff
@@ -462,6 +479,15 @@ void testMain() {
       const MethodCall show = MethodCall('TextInput.show');
       sendFrameworkMessage(codec.encodeMethodCall(show));
 
+      // The "setSizeAndTransform" message has to be here before we call
+      // checkInputEditingState, since on some platforms (e.g. Desktop Safari)
+      // we don't put the input element on DOM until we get its correct
```

Comment:
Nit: "on DOM" => "into the DOM"

If that's correct and I'm not misunderstanding, I think it's more explicit this way.

Reply:
Same for the duplicate comments below.